yellowbrick/features/manifold.py

```diff
@@ -236,16 +239,21 @@ def manifold(self, transformer):
         """
         if not is_estimator(transformer):
             if transformer not in self.ALGORITHMS:
-                raise YellowbrickValueError(
-                    "could not create manifold for '%s'".format(str(transformer))
+                    raise YellowbrickValueError("could not create manifold for '{}'".format(str(transformer))
                 )
 
+            if (transformer not in ('tsne','mds','hessian') and self.n_neighbors is None):
+                self.n_neighbors = 5
+                warnmsg = "using n_neighbors={}; please explicitly pass this argument for the '{}' manifold".format(self.n_neighbors, str(transformer))
+                warnings.warn(warnmsg, YellowbrickWarning)
```

Comment:
Thank you for adding the warning here! However, I think we should be more explicit, e.g.

```python
# 2 components is required for 2D plots, could optionally make it three for 3D plots in future
n_components = 2

requires_default_neighbors = {'lle', 'ltsa', 'isomap', 'hessian', 'spectral', 'modified'}
if self.n_neighbors is None and transformer in requires_default_neighbors:
    if transformer == 'hessian':
        self.n_neighbors = 1 + (n_components * (1 + (n_components + 1) / 2))
    else:
        self.n_neighbors = 5 

    warnmsg = "using n_neighbors={}; please explicity specify for the '{}' manifold".format(self.n_neighbors, transfomrer)
    warnings.warn(warnmsg, YellowbrickWarning)


```